### PR TITLE
adding policy dialog

### PR DESCRIPTION
**Pull request by alexjavabraz**
- Private Policy content added
- all styles are used only to display this content based on the document received from legal, that's why the CSS content in inline internally on the file.

### CODE REVIEW

File: src/common/views/Home.vue [hunk 1]
```diff
@@ -65,13 +65,13 @@
             </a>
           </span>
         </v-row>
-        <v-row class="d-flex justify-center gc-2 align-baseline flex-wrap">
-          <span>To learn about the various RBTC Access Methods, visit the</span>
+        <div class="d-flex justify-center gc-2 align-baseline flex-wrap">
+          <span>To learn about the varios RBTC access methos, visit</span>
```
Comment (lserra-iov) [inline src/common/views/Home.vue:69]: Looks like a typo

File: src/common/components/layouts/Footer.vue [hunk 2]
```diff
@@ -31,21 +32,27 @@
         :icon="mdiDiscord">
       </v-btn>
     </div>
+    <!-- eslint-disable-next-line max-len -->
```
Comment (lserra-iov) [inline src/common/components/layouts/Footer.vue:35]: I don't think this is necessary
```suggestion

```

File: src/common/components/common/PolicyDialog.vue [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+                <p style="text-align: justify;"><strong><span style="font-size:10pt;">Withdrawal of consent</span></strong></p>
+                <p style="text-align: justify;"><span style="font-size:10pt;">You may withdraw consent previously given by you to us at any time. However, this will not affect the lawfulness of any processing carried out before you withdraw your consent. If you withdraw your consent, we may not be able to provide certain products or services to you. We will advise you if this is the case at the time you withdraw your consent.</span></p>
+                <p style="text-align: justify;"><strong><u><span style="font-size:10pt;">CONTACT DETAILS</span></u></strong></p>
+                <p style="text-align: justify;"><span style="font-size:10pt;">If you have any queries regarding this Privacy Policy, if you wish to exercise any of your rights set out above or if you think that the Privacy Policy has not been followed, please contact us by emailing at&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:#1155cc;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.</span></p>
```
Comment (annipi) [inline src/common/components/common/PolicyDialog.vue:168]:
```suggestion
                <p style="text-align: justify;"><span style="font-size:10pt;">If you have any queries regarding this Privacy Policy, if you wish to exercise any of your rights set out above or if you think that the Privacy Policy has not been followed, please contact us by emailing at&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:white;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.</span></p>
```

File: src/common/components/common/PolicyDialog.vue [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+                    </li>
+                </ul>
+                <p style="text-align: justify;"><span style="font-size:10pt;">Please refer to your device&rsquo;s help material to learn what controls you can use to remove cookies, block cookies, or disable IDFA tracking. If you do this, it may affect your use of the Services.</span></p>
+                <p style="text-align: justify;"><span style="font-size:10pt;">We may also partner with third parties who may collect anonymous usage or statistical data through your use of the Services (including, for example, sub-contractors in technical, payment and/or delivery services, business partners, advertising networks, analytics providers).</span></p>
+                <p style="text-align: justify;"><strong><span style="font-size:10pt;">Updating your information</span></strong></p>
+                <p style="text-align: justify;"><span style="font-size:10pt;">If you want to update the information you have previously given to us, you can contact us at&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:#1155cc;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.&nbsp;</span></p>
```
Comment (annipi) [inline src/common/components/common/PolicyDialog.vue:75]:
```suggestion
                <p style="text-align: justify;"><span style="font-size:10pt;">If you want to update the information you have previously given to us, you can contact us at&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:white;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.&nbsp;</span></p>
```

File: src/common/components/common/PolicyDialog.vue [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+                        <p style="text-align: justify;"><span style="font-size:10pt;">to a country recognised by Uruguay or the European Commission as providing an adequate level of protection; or</span></p>
+                    </li>
+                    <li style="list-style-type:disc;font-size:10pt;">
+                        <p style="text-align: justify;"><span style="font-size:10pt;">to a country that does not offer adequate protection but whose transfer has been governed by the standard contractual clauses of the applicable law or by implementing other appropriate cross-border transfer solutions to provide adequate protection.</span></p>
+                    </li>
+                </ul>
+                <p style="text-align: justify;"><span style="font-size:10pt;">In the event that personal information is submitted, you agree to this transfer, storing or processing. If you would like more information about the mechanism via which your personal data is transferred please contact&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:#1155cc;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.&nbsp;</span></p>
```
Comment (annipi) [inline src/common/components/common/PolicyDialog.vue:146]:
```suggestion
                <p style="text-align: justify;"><span style="font-size:10pt;">In the event that personal information is submitted, you agree to this transfer, storing or processing. If you would like more information about the mechanism via which your personal data is transferred please contact&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:white;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.&nbsp;</span></p>
```

File: src/common/components/common/PolicyDialog.vue [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+                    </li>
+                    <li style="list-style-type:disc;font-size:10pt;">
+                        <p style="text-align: justify;"><strong><u><span style="font-size:10pt;">Object Processing</span></u></strong><span style="font-size:10pt;">: You may object to processing of your personal data where we rely on legitimate interest for processing that personal data. We will comply with your request unless we have a compelling overriding legitimate interest for processing or we need to continue processing your personal data to establish, exercise or defend a legal claim.&nbsp;</span></p>
+                    </li>
+                </ul>
+                <p style="text-align: justify;">
+                    <span style="font-size:10pt;">To exercise any of the rights listed above and/or any other right stated by the applicable law, you can contact us to&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:#1155cc;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">. We will immediately process your request and inform you the results within the term established by the applicable law.&nbsp;</span></p>
```
Comment (annipi) [inline src/common/components/common/PolicyDialog.vue:164]:
```suggestion
                    <span style="font-size:10pt;">To exercise any of the rights listed above and/or any other right stated by the applicable law, you can contact us to&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:white;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">. We will immediately process your request and inform you the results within the term established by the applicable law.&nbsp;</span></p>
```

File: src/common/components/common/PolicyDialog.vue (hunk 3 above)
Comment (annipi) [inline src/common/components/common/PolicyDialog.vue:168]:
```suggestion
                <p style="text-align: justify;"><span style="font-size:10pt;">If you have any queries regarding this Privacy Policy, if you wish to exercise any of your rights set out above or if you think that the Privacy Policy has not been followed, please contact us by emailing at&nbsp;</span><a href="mailto:legal@rootstocklabs.com"><u><span style="color:white;font-size:10pt;">legal@rootstocklabs.com</span></u></a><span style="font-size:10pt;">.</span></p>
```

**Review by annipi — COMMENTED** [review]
I left some suggestions to change this
Screenshot 2024-07-24 at 3 30 16 PM
to this
Screenshot 2024-07-24 at 3 36 02 PM

**Review by lserra-iov — APPROVED** [review]

**Review by annipi — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**